docs/diagrams/ProgressCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,96 @@
+@startuml
+!include style.puml
+
+title AddCommand\n
```

Comment:
```suggestion
title ProgressCommand\n
```

I think this might be a type

Reply:
Oh right thanks for pointing this out! Must be an error when I was copy pasting code across files haha 😄